tests/functional/DatadirTest.php

```diff
@@ -0,0 +1,437 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\SnowflakeTransformation\DatadirTests;
+
+use Keboola\DatadirTests\AbstractDatadirTestCase;
+use Keboola\DatadirTests\DatadirTestSpecification;
+use Keboola\SnowflakeDbAdapter\Connection;
+use Keboola\SnowflakeDbAdapter\QueryBuilder;
+use Keboola\SnowflakeTransformation\Config;
+use Keboola\SnowflakeTransformation\ConfigDefinition;
+use Symfony\Component\Process\Process;
+
+class DatadirTest extends AbstractDatadirTestCase
+{
+    protected Connection $db;
+
+    public function testTransformData(): void
+    {
+        // phpcs:disable Generic.Files.LineLength
+        $configArray = [
+            'authorization' => $this->getDatabaseConfig(),
+            'parameters' => [
+                'blocks' => [
+                    [
+                        'name' => 'first block',
+                        'codes' => [
+                            [
+                                'name' => 'first code',
+                                'script' => [
+                                    'DROP TABLE IF EXISTS "output"',
+                                    'CREATE TABLE IF NOT EXISTS "output" ("name" VARCHAR(200),"usercity" VARCHAR(200));',
+                                    "INSERT INTO \"output\" VALUES ('ondra', 'liberec'), ('odin', 'brno'), ('najlos', 'liberec')",
+                                ],
+                            ],
+                        ],
+                    ],
+                ],
+            ],
+        ];
+        // phpcs:enable
+
+        $this->runAppWithConfig($configArray);
```

Comment:
v com se to lisi od runTestWithCustomConfiguration ?

Reply:
tím že tam vracím Process

Reply:
no stejne nevim proc to delas? prijde mi, ze akorat kontrolujes vystup z toho procesu a to umi runTestWithCustomConfiguration taky, ne?
Jinak pokud je to jen kvuli tomu, tak PR na datadir, at to vraci - jestli je k tomu nejakej usecase, tak myslim neni problem, aby to vracelo process.